### ISSUE TITLE
Make Data Entry the primary tab, Reference Secondary

### ISSUE THREAD

**Issue by beckbria** [beckbria/puzztool#66]
When someone goes to Resistor/Semaphore/Braille/Morse, they should be taken to the encoding entry tool, not the reference.  Change the tab order.
